scene/open-mobile-scene-package/src/main/java/com/esri/samples/open_mobile_scene_package/OpenMobileScenePackageSample.java

```diff
@@ -53,56 +49,15 @@ public void start(Stage stage) throws Exception {
     // create a new scene view
     sceneView = new SceneView();
 
-    // load a mobile scene package
+    // create a mobile scene package from a file
     final String mspkPath = new File("./samples-data/mspk/philadelphia.mspk").getAbsolutePath();
```

Comment:
Maybe this one got missed. It needs the `System.getProperty("data.dir")` parent directory argument.

Reply:
I thought that's a change we're only doing in `dev` for now. 
Should I target this PR at `dev` instead?

Reply:
You're right. Thought it was targeting dev. Disregard.